javascriptv3/example_code/lambda/guide_supplements/guide-cdk-lambda-function.js

```diff
@@ -1,28 +1,35 @@
-/*
+mat/*
```

Comment:
is this on purpose

Reply:
Nope. :) I'm following up with some unit tests, I'll grab this as well.